libs/server/src/lib/utils/utils.ts

```diff
@@ -169,14 +169,17 @@ export async function normalizeSchema(
 
```

Comment:
Change to make the property required if there is an x-prompt with no default,
and a little clean up to a) set up the basic option then b) check the xPrompt to set items and tooltips